src/status_im/contexts/settings/language_and_currency/data_store.cljs

```diff
@@ -0,0 +1,32 @@
+
+(ns status-im.contexts.settings.language-and-currency.data-store
+  (:require [clojure.set :as set]
+            [utils.i18n :as i18n]))
+
+(defn rpc->currency
+  [currency]
+  (-> currency
+      (set/rename-keys
+       {:shortName :code
```

Comment:
we have a static currencies list and replaced with this. i will check for usage and if not too much, will change it to `:short-name`

Reply:
As I said before, if it is an existing rename let's keep it, since finding all the usages may lead to bugs. If it's a new one rename added, then it's better to pick something closer to the original name.